app/policies/lighthouse_policy.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+LighthousePolicy = Struct.new(:user, :lighthouse) do
+  def access_claims?
+    if user.participant_id.present?
+      log_success('lighthouse_claims')
+    else
+      log_failure('lighthouse_claims')
+    end
+  end
+
+  private
+
+  def log_success(policy)
+    StatsD.increment('api.lighthouse.benefits_claims.policy.success', tags: ["policy:#{policy}"]) if user.loa3?
+    true
+  end
+
+  def log_failure(policy)
+    StatsD.increment('api.lighthouse.benefits_claims.policy.failure', tags: ["policy:#{policy}"]) if user.loa3?
```

Comment:
you are only logging (both success and failiures) if the user is loa3? Is that intentional?

Reply:
Full transparency I copy-paste-replaced what was in the `app/policies/evss_policy.rb` because Lighthouse has slightly different requirements. It's possible that we don't the `if` part, but I don't think it will hurt for the moment